### ISSUE TITLE
Get page content with Elementor divs

### ISSUE THREAD

**Issue by nicolas92** [pojome/elementor#776]
Hello, I've a question, is it be possible to have the contents of a page created with elementor?

I created an option in the customizer to display a page instead of widgets for the footer.
I created a "Footer" page with Elementor, on the front end of the page, the Elementor widgets are displayed perfectly in the footer but in all other pages, there is no Elementor wrap divs, only the content of the page.
It is possible to display the content of a page with the Elementor wrap divs via the id of the page?

I tried the following two codes:

```
$page_id = get_theme_mod( 'footer_page_id' );
$page = get_post( $page_id );

echo apply_filters( 'the_content', $page->post_content )

// the second code
echo apply_filters( 'elementor/frontend/the_content', $page->post_content );
```

Thank you a lot.

**Comment by arielk** [comment]
We're working on a similar feature, it will be released soon. So I merged this issue with original issue #496 that our Dev staff could deal with it more efficiently. So please let's continue the conversation there.

**Comment by nicolas92** [comment]
Hello, okay, thank you very much :)
